clustersConfig.py

```diff
@@ -141,6 +141,7 @@ def current_host() -> str:
 class ClustersConfig:
     name: str
     kubeconfig: str
+    dpu_operator_pull_number: Optional[str] = None
```

Comment:
We should pass arguments into the extra config instead of polluting the global cluster config.

Reply:
okay. I will fix that